test/integration/YieldTokenIncidents/submitClaim.js

```diff
@@ -0,0 +1,219 @@
+const { ethers } = require('hardhat');
+const { constants: { ZERO_ADDRESS } } = require('@openzeppelin/test-helpers');
+const { expect } = require('chai');
+
+const { daysToSeconds } = require('../../unit/IndividualClaims/helpers');
+
+const { mineNextBlock, setNextBlockTime } = require('../../utils/evm');
+const { BigNumber } = require('ethers');
+
+const { parseEther } = ethers.utils;
+
+const setTime = async timestamp => {
+  await setNextBlockTime(timestamp);
+  await mineNextBlock();
+};
+
+const priceDenominator = '10000';
+
+describe.skip('submitClaim', function () {
```

Comment:
is this supposed to be skipped? if yes, I assume it'll be addresses in https://github.com/NexusMutual/smart-contracts/pull/267

Reply:
correct